app/src/main/res/layout/list_circle_try.xml

```diff
@@ -48,5 +48,15 @@
             android:layout_height="wrap_content" />
     </RelativeLayout>
 
+    <ImageView
+        android:id="@+id/toDoListItemDoneImage"
+        android:gravity="center"
+        android:layout_weight="2"
+        android:layout_width="0dp"
+        android:layout_height="35dp"
```

Comment:
extract hardcoded values like `35dp` to `dimens.xml` or use existing. Code woulb easier to read and maintain.